Havit.Blazor.Components.Web.Bootstrap/Grids/HxGrid.razor

```diff
@@ -254,7 +254,9 @@
 				<HxPager TotalPages="@totalPages"
 						 CurrentPageIndex="@CurrentUserState.PageIndex"
 						 CurrentPageIndexChanged="@HandlePagerCurrentPageIndexChanged"
-                         CssClass="@PagerCssClassEffective" />
+                         CssClass="@PagerCssClassEffective"
```

Comment:
As we are releasing a new major version of Havit.Blazor I will remove the PagerCssClass parameter + related.